### DISCUSSION TITLE
Linux license question

### DISCUSSION THREAD

**Discussion by dggbg** [airsdk/Adobe-Runtime-Support#2573] [General]
Hello, 
is there a chance for licensing model changes, like: monthly payments or personal license (with splash screen as an example), or other options for projects that are not for sale. 

Personally i just want to migrate, wine is just fine (in my case) but i decided to ask anyway.
Thank you, Best Regards!

**Comment by rjp421** [comment]
@ajwfrost is it possible to get a license for testing on linux, please?

i am disabled and without income, and only want to test out old/new projects on Linux, both PC (x86_64) and eventually RaspberryPi 3/4/5 (ARM64).

thanks for helping keep AIR alive.

**Comment by punkice3407** [comment]
same
